packages/codemods/src/codemods/v2.x.x/configureFastify/configureFastify.ts

```diff
@@ -0,0 +1,30 @@
+import type { FileInfo, API, Identifier, MemberExpression } from 'jscodeshift'
+
+export default function transform(file: FileInfo, api: API) {
+  const j = api.jscodeshift
+  const ast = j(file.source)
+
+  ast.find(j.ExpressionStatement).forEach((path) => {
```

Comment:
Bit of a nitpick, so ok if you want to dismiss, but you could just do j.AssignmentExpression here... or even better if we used `j.ExportDeclaration

Reply:
Realised j.ExportDeclaration won't work here, but worth a check if we can make it more specific